config.json

```diff
@@ -205,6 +206,17 @@
           "strings"
         ]
       },
+      {
+        "slug": "anagram",
+        "name": "Anagram",
+        "uuid": "451d5472-daad-47a3-b3cd-41fa9b775c79",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 1,
+        "topics": [
+          "strings"
+        ]
```

Comment:
I don't know if this is your intent, but the `topics` field is basically only there for v2 tracks to help them determine the `practices` and `prerequisites` values. For a new track like Wren, I would suggest to just populate the `practices` and `prerequisites` fields. See https://github.com/exercism/docs/blob/main/building/tracks/config-json.md#practice-exercises

Reply:
Are these things not surfaced in the UI like they have been in the past?  Ah.  I think I'm using topics in place of `practices` so I just need to rename them all...

Jeremy told me I can't use `prerequisites` in practice exercises (IIRC).